targets/targets.json

```diff
@@ -3033,6 +3033,48 @@
         ],
         "device_name": "STM32L072CZ"
     },
+    "STM32L071CXCTX": {
+        "inherits": [
+            "MCU_STM32"
+        ],
+        "core": "Cortex-M0+",
+        "public": false,
+        "extra_labels_add": [
+            "STM32L0",
+            "STML071CxCTx",
+            "STML071xx"
+        ],
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC (need HW patch) | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI | USE_PLL_MSI",
+                "value": "USE_PLL_MSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            }
+        },
+        "macros_add": [
+            "MBED_TICKLESS",
+            "EXTRA_IDLE_STACK_REQUIRED"
+        ],
+        "overrides": {
+            "lpticker_delay_ticks": 0
+        },
+        "device_has_add": [
+            "CRC",
+            "SERIAL_ASYNCH",
+            "FLASH",
+            "MPU"
+        ],
+        "bootloader_supported": true,
+        "device_name": "STM32L071CXCTX",
+        "mbed_rom_start": "0x8000000",
+        "mbed_rom_size": "0x30000",
+        "mbed_ram_start": "0x20000000",
+        "mbed_ram_size": "0x5000"
```

Comment:
Should be not useful if device_name is well set

Reply:
ok, I had "mbed-cli" complaining because it didn't have this, does it serve a purpose?